src/main/features/response/routes/response-type.ts

```diff
@@ -55,7 +57,11 @@ export default express.Router()
             res.redirect(Paths.sendYourResponseByEmailPage.evaluateUri({ externalId: claim.externalId }))
             break
           case ResponseType.FULL_ADMISSION:
-            res.redirect(Paths.sendYourResponseByEmailPage.evaluateUri({ externalId: claim.externalId }))
+            if (toBoolean(config.get<boolean>('featureToggles.fullAdmission'))) {
```

Comment:
Please use the `FeatureToggles` class instead of using config directly

Reply:
It should work, how did you use it?

Reply:
FeatureToggleGuard.anyFeatureEnabledGuard('fullAdmission')